### ISSUE TITLE
how to create a record ?

### ISSUE THREAD

**Issue by dfang** [OCA/odoorpc#23]
how to create a record by odoorpc so that importing records by code ?

thanks !

**Comment by sebalix** [comment]
What do you mean by "code"? Get a product from its `default_code` for instance?

**Comment by dfang** [comment]
no, on documentation, browsing and updating a record with odooprc is very easy, i'm not sure if we can create a record by odoorpc ?

**Comment by sebalix** [comment]
Just use the `create` method from the any model proxy:
```python
Product = odoo.env['product.product']
product_id = Product.create({...})
product = Product.browse(product_id)
```
https://odoorpc.readthedocs.io/en/latest/tuto_browse_methods.html

Is it what you are looking for?
